charts/veritable-ui/values.yaml

```diff
@@ -0,0 +1,643 @@
+## @section Global parameters
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry, imagePullSecrets and storageClass
+##
+
+## @param global.imageRegistry Global Docker image registry
+## @param global.imagePullSecrets Global Docker registry secret names as an array
+## @param global.storageClass Global StorageClass for Persistent Volume(s)
+##
+global:
+  imageRegistry: ""
+  ## E.g.
+  ## imagePullSecrets:
+  ##   - myRegistryKeySecretName
+  ##
+  imagePullSecrets: []
+  storageClass: ""
+
+## @section Common parameters
+##
+
+## @param kubeVersion Override Kubernetes version
+##
+kubeVersion: ""
+## @param nameOverride String to partially override common.names.name
+##
+nameOverride: ""
+## @param fullnameOverride String to fully override common.names.fullname
+##
+fullnameOverride: ""
+## @param namespaceOverride String to fully override common.names.namespace
+##
+namespaceOverride: ""
+## @param commonLabels Labels to add to all deployed objects
+##
+commonLabels: {}
+## @param commonAnnotations Annotations to add to all deployed objects
+##
+commonAnnotations: {}
+## @param clusterDomain Kubernetes cluster domain name
+##
+clusterDomain: cluster.local
+## @param extraDeploy Array of extra objects to deploy with the release
+##
+extraDeploy: []
+
+## Enable diagnostic mode in the deployment
+##
+diagnosticMode:
+  ## @param diagnosticMode.enabled Enable diagnostic mode (all probes will be disabled and the command will be overridden)
+  ##
+  enabled: false
+  ## @param diagnosticMode.command Command to override all containers in the deployment
+  ##
+  command:
+    - sleep
+  ## @param diagnosticMode.args Args to override all containers in the deployment
+  ##
+  args:
+    - infinity
+
+## @section veritable cloudagent config parameters
+## TODO update name to companys
+## TODO bash completion make sure kubectl (bash complete@2)
+companysHouseApiKey:
+  enabled: true
+  secret: "companysHouseApiKey"
+  existingSecret: ""
+  existingSecretKey: ""
+
+companysHouseApiUrl: "https://api.company-information.service.gov.uk"
+logLevel: "debug"
+
+cookieSessionKeys:
+  enabled: true
+  secret:
+    - "secret"
+  existingSecret: ""
+  existingSecretKey: ""
+
+publicUrl: "http://localhost:3000"
+apiSwaggerBgColor: "#fafafa"
+apiSwaggerTitle: "Veritable"
+apiSwaggerHeading: "Veritable"
+
+idpClientId: "veritable-ui"
+idpPublicURLPrefix: "http://localhost:3080/realms/veritable/protocol/openid-connect"
+idpInternalURLPrefix: "http://localhost:3080/realms/veritable/protocol/openid-connect"
+idpAuthPath: "/auth"
+idpTokenPath: "/token"
+idpJWKSPath: "/certs"
```

Comment:
For all the values you have added, you need to add the [bitnami readme generator](https://github.com/bitnami/readme-generator-for-helm?tab=readme-ov-file#valuesyaml-metadata) to this file and ensure that they make sense.

Reply:
on it! thanks